src/test/e2e/e2e_helm_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2eExampleHelm(t *testing.T) {
+	//defer e2e.cleanupAfterTest(t)
```

Comment:
Can you uncomment this deferred cleanup call :)

All of our test files are run sequentially on the same cluster on whatever host runner GitHub gives us. Since they're all running on the same cluster after tests that modify the cluster (doing an init and deploying a package like we are in this test) we defer this cleanup to get the cluster to a 'fresh' state. 

Without this cleanup call, depending on the order the `go test ./...` command runs the tests, the test that gets run immediately after this one will fail because you cannot `zarf init` on the same cluster twice currently.

Reply:
Yes, oops! I meant to put that back before committing.